src/core/public/chrome/nav_group/nav_group_service.test.ts

```diff
@@ -437,41 +437,8 @@ describe('ChromeNavGroupService#start()', () => {
       .pipe(first())
       .toPromise();
 
-    expect(currentNavGroup?.id).toEqual('bar-group');
-    expect(currentNavGroup?.title).toEqual('barGroupTitle');
-  });
-
-  it('should be able to find the right nav group when visible nav group length is 1 and is not all nav group', async () => {
-    const uiSettings = uiSettingsServiceMock.createSetupContract();
-    const navGroupEnabled$ = new Rx.BehaviorSubject(true);
-    uiSettings.get$.mockImplementation(() => navGroupEnabled$);
-
-    const chromeNavGroupService = new ChromeNavGroupService();
-    const chromeNavGroupServiceSetup = chromeNavGroupService.setup({ uiSettings });
-
-    chromeNavGroupServiceSetup.addNavLinksToGroup(
-      {
-        id: 'foo',
-        title: 'fooGroupTitle',
-        description: 'foo description',
-      },
-      [mockedNavLinkFoo]
-    );
-
-    const chromeNavGroupServiceStart = await chromeNavGroupService.start({
-      navLinks: mockedNavLinkService,
-      application: mockedApplicationService,
-      breadcrumbsEnricher$: new Rx.BehaviorSubject<ChromeBreadcrumbEnricher | undefined>(undefined),
-      workspaces: workspacesServiceMock.createStartContract(),
-    });
-    mockedApplicationService.navigateToApp(mockedNavLinkFoo.id);
-
-    const currentNavGroup = await chromeNavGroupServiceStart
-      .getCurrentNavGroup$()
-      .pipe(first())
-      .toPromise();
-
-    expect(currentNavGroup?.id).toEqual('foo');
+    expect(currentNavGroup?.id).toEqual(undefined);
+    expect(currentNavGroup?.title).toEqual(undefined);
```

Comment:
The test description said `set current nav group automatically if application only belongs to 1 visible nav group`, but the `currentNavGroup` is empty currently. Do we need to update the test description or test body?

Reply:
Sure, updated.